akka-actor/src/main/scala/akka/pattern/BackoffOnRestartSupervisor.scala

```diff
@@ -81,6 +79,15 @@ private class BackoffOnRestartSupervisor(
       stop(self)
   }
 
-  def receive = onTerminated orElse handleBackoff
+  def receive: Receive = onTerminated orElse handleBackoff
```

Comment:
Thank you! Huge nit for me when these are not explicit. `Actor.Receive` is much nicer IMO.

Reply:
In general I prefer for public methods to have an explicit type